src/lib.rs

```diff
@@ -1488,14 +1489,23 @@ impl Library {
         let c_filename = CString::new(library_filename)?;
         let mut status = Status::new();
         let inner = unsafe { tf::TF_LoadLibrary(c_filename.as_ptr(), status.inner()) };
+
+        let buf = unsafe {
+            let mut buf = tf::TF_GetOpList(inner);
```

Comment:
`TF_GetOpList` cannot be called with NULL, so `inner` needs to be checked first.

Reply:
I've moved the `TF_GetOpList` call to after the `inner.null()` check